test/integration/tarantool_role_test.lua

```diff
@@ -0,0 +1,172 @@
+local t = require('luatest')
+local fio = require('fio')
+local g = t.group('tarantool_role_test')
+local fiber = require('fiber')
+local fun   = require('fun')
+local net_box = require('net.box')
+
+local iter = fun.iter
+
+local Server = require('test.integration.server')
+
+local tmpdir = fio.tempdir()
+
+local function split(self, sep)
+    local fields = {}
+    local pattern = string.format("([^%s]+)", sep)
+    self:gsub(pattern, function(c) table.insert(fields, c) end)
+    return fields
+end
+
+local function reducer(res, l, r)
+    if res ~= nil then
+        return res
+    end
+    if tonumber(l) == tonumber(r) then
+        return nil
+    end
+    return tonumber(l) > tonumber(r)
+end
+
+--- e.g
+--- input: "2.11.0"
+--- output: {"2", "11", "0"}
+local function split_version(version_string)
+    local vtable  = split(version_string, '.')
+    local vtable2 = split(vtable[3],  '-')
+    vtable[3], vtable[4] = vtable2[1], vtable2[2]
+    return vtable
+end
+
+local function isTarantoolVersionGreater(expected, version)
+    version = version or _TARANTOOL
+    if type(version) == 'string' then
+        version = split_version(version)
+    end
+    local res = iter(version):zip(expected):reduce(reducer, nil)
+    if res or res == nil then res = true end
+    return res
+end
+
+g.before_all(function (cg)
+    if not isTarantoolVersionGreater({3, 0, 0}) then
+        t.skip('Tarantool role is supported only for Tarantool starting from v3.0.0')
+    end
+
+    cg.router = Server:new({
+        config_file = fio.abspath(fio.pathjoin('test', 'integration', 'simple_app', 'config.yaml')),
+        env = {rocks_dir = fio.abspath('.')},
+        chdir = fio.pathjoin(tmpdir, 'simple_app'),
+        alias = 'master',
+        workdir = fio.pathjoin(tmpdir, 'simple_app'),
+    })
+
+    print(tmpdir)
+
+end)
+
+g.before_each(function(cg)
+    fio.mktree(cg.router.workdir)
+    fio.copytree(fio.abspath('./.rocks'), fio.pathjoin(cg.router.workdir, '.rocks'))
+    cg.router:start{wait_until_ready = true}
+    fio.rmtree(tmpdir)
+end)
+
+g.after_each(function(cg)
+    cg.router:stop()
+    fio.rmtree(tmpdir)
+end)
+
+g.test_simple_role_config = function()
+    local conn = net_box.connect('localhost:3313')
+
+    conn:eval([[
```

Comment:
I think somewhere in the object we already have a connection.
```suggestion
   cg.router.net_box:eval([[
```